packages/base-py/pluto_base/utils/resource_id.py

```diff
@@ -0,0 +1,30 @@
+import hashlib
+from .configuration import current_project_name, current_stack_name
+
+RESOURCE_ID_MAX_LENGTH = 64
+
+
+def gen_resource_id(*args):
```

Comment:
What does `*args` mean?

Reply:
There are two ways to utilize this function, each requiring a different number of arguments. This function has been directly translated from TypeScript, available at https://github.com/pluto-lang/pluto/blob/main/packages/base/utils/resource-id.ts . I'll include comments to explain the use of `*args`.

Reply:
Make sense.